src/components/Clients/ClientListTable.jsx

```diff
@@ -0,0 +1,87 @@
+// React Imports
+import React, { useContext } from 'react';
+// Material UI Imports
+import { styled, useTheme } from '@mui/material/styles';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell, { tableCellClasses } from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+// Custom Hook Imports
+import { useStatusNotification } from '../../hooks';
+// Context Imports
+import { UserListContext } from '../../contexts';
+// Component Imports
+import { StatusNotification } from '../Notification';
+import ClientListTableRow from './ClientListTableRow';
+
+/**
+ * ClientListTable Component - Component that generates table of clients from data within ClientList
+ *
+ * @memberof Clients
+ * @name ClientListTable
+ */
+
+const ClientListTable = ({ loadingActive, statusType, defaultMessage }) => {
```

Comment:
The loadingActive state in Line 27 and 52 is no longer in use. Can be removed without affecting it.